applications/matlab-ls6/app.json

```
@@ -0,0 +1,104 @@
+        "SINGULARITY_RUN"
+    ],
+    "containerImage": "docker://taccaci/interactive-base:1.1.0",
+    "jobType": "BATCH",
+    "maxJobs": -1,
+    "maxJobsPerUser": -1,
+    "strictFileInputs": true,
+    "jobAttributes": {
+        "description": "Run an interactive MATLAB 2022b session on LoneStar6. Submit a help ticket to get your license configured or approved.",
+        "dynamicExecSystem": false,
+        "execSystemConstraints": null,
+        "execSystemId": "ls6",
+        "execSystemExecDir": "${JobWorkingDir}",
+        "execSystemInputDir": "${JobWorkingDir}",
+        "execSystemOutputDir": "${JobWorkingDir}/output",
+        "execSystemLogicalQueue": "development",
+        "archiveSystemId": "cloud.data",
+        "archiveSystemDir": "HOST_EVAL($HOME)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}",
+        "archiveOnAppError": true,
+        "isMpi": false,
+        "mpiCmd": null,
+        "cmdPrefix": null,
+        "parameterSet": {
+            "appArgs": [],
+            "containerArgs": [
+                {
+                    "name": "Interactive Session and TACC Module Mounts",
+                    "description": "Mount the required folders in order for TAP, DCV, VNC, and TACC modules to function.",
+                    "inputMode": "FIXED",
+                    "arg": "--bind /bin,/lib,/lib64,/share,/run,/opt/apps,/usr/lib,/usr/lib64,/usr/libexec,/usr/bin,/usr/etc,/usr/include,/usr/share,/usr/sbin,/usr/src",
+                    "notes": {
+                        "isHidden": true
+                    }
+                }
+            ],
+            "schedulerOptions": [
+                {
+                    "name": "MATLAB TACC Scheduler Profile",
+                    "description": "Scheduler profile for TACC MATLAB R2022b.",
+                    "inputMode": "FIXED",
+                    "arg": "--tapis-profile matlab_2022b_ls6",
+                    "notes": {
+                        "isHidden": true
+                    }
+                },
+                {
+                    "name": "TACC Interactive Session Substrings",
+                    "description": "VNC and DCV sessions require the substrings 'tap_' and '-dcvserver' in the slurm job name in order to function.",
+                    "inputMode": "FIXED",
+                    "arg": "--job-name ${JobName}-dcvserver-tap_",
+                    "notes": {
+                        "isHidden": true
+                    }
+                }
+            ],
+            "envVariables": [
+                {
+                    "key": "_XTERM_CMD",
+                    "value": "swr -t 8 matlab",
+                    "description": "Command passed to XTERM, launched within the interactive session."
+                },
+                {
+                    "key": "APPEND_PATH",
+                    "value": "$PATH",
+                    "description": "Append host $PATH to Singularity container default $PATH."
+                }
```

We can now set `inputMode` and `notes` on the `envVariables` field!

```suggestion
                {
                    "key": "_XTERM_CMD",
                    "value": "swr -t 8 matlab",
                    "description": "Command passed to XTERM, launched within the interactive session.",
                    "inputMode": "FIXED",
                    "notes": {
                        "isHidden": true
                    }
                },
                {
                    "key": "APPEND_PATH",
                    "value": "$PATH",
                    "description": "Append host $PATH to Singularity container default $PATH.",
                    "inputMode": "FIXED",
                    "notes": {
                        "isHidden": true
                    }
                }
```

---

Great, I committed the changes!